src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj

```diff
@@ -10,17 +10,17 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Fody" Version="[2.*,3)" PrivateAssets="All" />
-    <PackageReference Include="GitVersionTask" Version="[4.0.0-*,5)" PrivateAssets="All" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0-*,8)" PrivateAssets="None" />
-    <PackageReference Include="Obsolete.Fody" Version="[4.*,5)" PrivateAssets="All" />
-    <PackageReference Include="Particular.CodeRules" Version="[0.*,2)" PrivateAssets="All" />
+    <PackageReference Include="Fody" Version="2.1.3" PrivateAssets="All" />
+    <PackageReference Include="GitVersionTask" Version="4.0.0-beta0012" PrivateAssets="All" />
+    <PackageReference Include="NServiceBus" Version="[7.0.0-beta0007,8)" PrivateAssets="None" />
```

Comment:
nitpick, but I'd prefer to have a space after the comma.

Reply:
I was going to say that no comma is canonical (the NuGet docs and samples have no comma) but then we do have a comma in the RabbitMQ.Client reference already, so we should be consistent, and I agree that a space makes the range easier to read. I'll add it.